modules/ROOT/pages/building-https-proxy.adoc

```diff
@@ -15,7 +15,7 @@ To configure your HTTPS API Proxy using secrets stored in your secret group, you
 * Secrets Manager Permissions enabled for your user. +
 To configure an existing TLS context for the API Proxy, you need the *Read secrets metadata* permission.
 Review the xref:anypoint-security::asm-permission-concept.adoc[Secrets Manager Permissions] reference for more information.
-* Make sure your user has *Manage API Configuration* role.
+* Make sure your user has the *Manage APIs Configuration* permission for the environment the proxy will be created in.
 * A secret group along with a TLS context in Secrets Manager. +
 Follow the xref:anypoint-security::asm-secret-group-creation-task.adoc[Create a Secret Group] instructions along with the  xref:anypoint-security::asm-secret-group-creation-task.adoc#add-a-tls-context[Add a TLS Context] task contained in that article if you need help creating a TLS context. +
```

Comment:
For adding a TLS context, the correct xref is:
xref:anypoint-security::secret-group-add-tls-context.adoc#add-tls-context[Add a TLS Context]